src/components/BottomBar/Origin/NodeNavigationButton.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {
+  MdCenterFocusStrong,
+  MdFlashOn,
+  MdFlight
+} from 'react-icons/md';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import Focus from 'svg-react-loader?name=Focus!../../../icons/focus.svg';
+
+import { NavigationAimKey, NavigationAnchorKey, RetargetAnchorKey } from '../../../api/keys';
+import InfoBox from '../../common/InfoBox/InfoBox';
+import Button from '../../common/Input/Button/Button';
+import Row from '../../common/Row/Row';
+import SvgIcon from '../../common/SvgIcon/SvgIcon';
+
+import styles from './NodeNavigationButton.scss';
+
+const NavigationTypes = {
+  jump: 'jump',
+  fly: 'fly',
+  focus: 'focus',
+  frame: 'frame'
+};
+
+function NavigationButton({
+  identifier, onFinish, showLabel, type
+}) {
+  const luaApi = useSelector((state) => state.luaApi);
+
+  function focus(event) {
+    if (!event.shiftKey) {
+      luaApi.setPropertyValueSingle(RetargetAnchorKey, null);
+    }
+    luaApi.setPropertyValueSingle(NavigationAnchorKey, identifier);
+    luaApi.setPropertyValueSingle(NavigationAimKey, '');
+    event.stopPropagation();
+    onFinish();
+  }
+
+  const flyTo = (event) => {
```

Comment:
I would go with one style of declaring functions. Personally i prefer "function" over "const" (I think we do this in most of the repo?)

Reply:
👍 Thanks, had missed this inconsistency